.ci/magician/cloudbuild/build_trigger.go

```diff
@@ -5,10 +5,10 @@ import (
 	"fmt"
 	"os"
 
-	"google.golang.org/api/cloudbuild/v1"
+	cloudbuildv1 "google.golang.org/api/cloudbuild/v1"
```

Comment:
this is not strictly necessary but I think it might be useful to disambiguate since this is inside the cloudbuild package.